pkg/core/interop_system.go

```diff
@@ -133,10 +129,7 @@ func putWithContextAndFlags(ic *interop.Context, stc *StorageContext, key []byte
 		return errors.New("StorageContext is read only")
 	}
 	si := ic.DAO.GetStorageItem(stc.ID, key)
-	if si != nil && si.IsConst {
-		return errors.New("storage item exists and is read-only")
-	}
-	sizeInc := 1
+	sizeInc := 0
```

Comment:
`len(si.Value) == 0` case is not handled correctly.

Reply:
Still there.

Reply:
Fixed.